pkg/kubelet/dockershim/docker_service.go

```diff
@@ -104,5 +115,10 @@ func (ds *dockerService) Version(_ string) (*runtimeApi.VersionResponse, error)
 }
 
 func (ds *dockerService) UpdateRuntimeConfig(runtimeConfig *runtimeApi.RuntimeConfig) error {
+	if ds.networkPlugin != nil && runtimeConfig.NetworkConfig.PodCidr != nil {
+		details := make(map[string]interface{})
+		details[network.NET_PLUGIN_EVENT_POD_CIDR_CHANGE_DETAIL_CIDR] = *runtimeConfig.NetworkConfig.PodCidr
```

Comment:
Yes, if we ever run kubelet and the shim as two separate processes, this will need to be persisted here, or individually by the network plugin packages. Right now, they restart together, so runtime will get the config again.